### PR TITLE
 [MIG] scheduler_error_mailer_therp

### PR DESCRIPTION
**Pull request by hbrunn**
migration itself lives in 2a07534

### CODE REVIEW

File: scheduler_error_mailer_therp/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Therp BV <http://therp.nl>
```
Comment (NL66278) [inline scheduler_error_mailer_therp/__manifest__.py]: https

File: scheduler_error_mailer_therp/hooks.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Therp BV <http://therp.nl>
```
Comment (NL66278) [inline scheduler_error_mailer_therp/hooks.py]: https

File: scheduler_error_mailer_therp/hooks.py [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import SUPERUSER_ID, api
```
Comment (NL66278) [inline scheduler_error_mailer_therp/hooks.py]: from odoo

File: scheduler_error_mailer_therp/models/__init__.py [hunk 4]
```diff
@@ -0,0 +1,4 @@
+# -*- coding: utf-8 -*-
+# © 2016 Therp BV <http://therp.nl>
```
Comment (NL66278) [inline scheduler_error_mailer_therp/models/__init__.py]: Copyright and https

File: scheduler_error_mailer_therp/models/ir_cron.py [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# -*- coding: utf-8 -*-
+# Copyright 2016-2018 Therp BV <http://therp.nl>
+# License AGPL-3.0 or later (http://www.gnu.org/licenses/agpl.html).
+from openerp import fields, models
```
Comment (NL66278) [inline scheduler_error_mailer_therp/models/ir_cron.py]: from odoo

**Review by NL66278 — CHANGES_REQUESTED** [review]
General code LGTM. Some nitpicking. See other comments.

**Comment by hbrunn** [comment]
thanks, pushed a rebased fixed version

**Review by NL66278 — APPROVED** [review]
:+1:  LGTM

**Pull request merged**